contributors/design-proposals/kubelet-eviction.md

```diff
@@ -425,6 +425,22 @@ from placing new best effort pods on the node since they will be rejected by the
 On the other hand, the `DiskPressure` condition if true should dissuade the scheduler from
 placing **any** new pods on the node since they will be rejected by the `kubelet` in admission.
 
+## Enforcing Node Allocatable
+
+To enforce [Node Allocatable](./node-allocatable.md), Kubelet primarily uses cgroups.
+However `storage` cannot be enforced using cgroups.
+
+Once Kubelet supports `storage` as an `Allocatable` resource, Kubelet will perform evictions whenever the total storage usage by pods exceed node allocatable.
+
+The trigger threshold for storage evictions will not be user configurable for the purposes of `Allocatable`.
+Kubelet will evict pods once the `storage` usage is greater than or equal to `98%` of `Allocatable`.
```

Comment:
I realize this is an implementation detail, but what are we trying to protect against by setting the threshold to 98%?  Why not 100%, since the overall node is protected by the eviction-hard threshold?